rust/xaynet-ffi/README.md

```diff
@@ -0,0 +1,18 @@
+# Xaynet FFI
+
+## Generate C-Header File
+
+`ffi-support` provides some helpful macros to reduce boilerplate code.
+However the feature (`--pretty=expanded`) to expand these macros during the generation of the
+C-header file is still unstable. Therefore we need to use the rust nightly.
+
+- run `RUSTUP_TOOLCHAIN=nightly cbindgen --output xaynet_ffi.h`
+
+
+## Run tests
+
+### macOS
+
+- run `cargo build`
+- run `cc -o tests/ffi_test.o -Wl,-dead_strip -I. tests/ffi_test.c ../target/debug/libxaynet_sdk.a -framework Security -framework Foundation`
```

Comment:
what are the `-framework Security -framework Foundation` arguments?

Reply:
when I try to run this:
```
cc: error: Security: No such file or directory
cc: error: Foundation: No such file or directory
cc: error: unrecognized command line option ‘-framework’
cc: error: unrecognized command line option ‘-framework’
```
I'm not on macOS. Is this macOS-specific?

Reply:
that is right, it is macOS-specific